cartographer/cloud/internal/map_builder_context_interface.h

```diff
@@ -43,6 +44,12 @@ class MapBuilderContextInterface : public async_grpc::ExecutionContext {
   // Calling with 'nullptr' signals subscribers that the subscription has ended.
   using LocalSlamSubscriptionCallback =
       std::function<void(std::unique_ptr<LocalSlamResult>)>;
+
+  // A return value of 'false' indicates that the subscription should end.
```

Comment:
Who signals whom? In the local slam case, server signals the subscribers by calling the callback with a nullptr argument.
Why is this different here? I think the bool should be an argument.

Reply:
As discussed made this more symmetric and added explaining comments.